rdkit/Chem/AtomPairs/Torsions.py

```diff
@@ -54,33 +54,27 @@ def pyScorePath(mol, path, size, atomCodes=None):
   >>> c3 = Utils.GetAtomCode(m.GetAtomWithIdx(2),2)
   >>> c4 = Utils.GetAtomCode(m.GetAtomWithIdx(4),1)
   >>> t = c1 | (c2 << rdMolDescriptors.AtomPairsParameters.codeSize) | (c3 << (rdMolDescriptors.AtomPairsParameters.codeSize*2)) | (c4 << (rdMolDescriptors.AtomPairsParameters.codeSize*3))
-  >>> pyScorePath(m,(0,1,2,4),4)==t
+  >>> pyScorePath(m, (0, 1, 2, 4), 4) == t
   1
 
   """
   codes = [None] * size
   for i in range(size):
-    if i == 0 or i == (size - 1):
-      sub = 1
-    else:
-      sub = 2
+    sub = 1 if i in (0, size - 1) else 2
```

Comment:
I personally find python's ternary syntax terrible, so I prefer the original form of this (a lot)
But since @gedeck didn't complain, I'm not going to insist.

Why do you think that this is better than what was previously there?